docker-compose.yml

```diff
@@ -1,6 +1,23 @@
 version: "3.9"
    
 services:
+
+  mysql:
+    image: mysql:5.7
+    restart: on-failure
+    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+    environment:
+      - MYSQL_ROOT_PASSWORD=cae_root_pw
+      - MYSQL_HOST=canvas_app_explorer_mysql
+      - MYSQL_PORT=3306
+      - MYSQL_DATABASE=canvas_app_explorer_local
+      - MYSQL_USER=cae_user
+      - MYSQL_PASSWORD=cae_pw
+    ports:
+      - "5306:3306"
```

Comment:
I'd change this 5306 to a different port, like maybe even 6306 is fine. This is really just so it doesn't conflit with our other local databases.